kratos/tests/utilities/run_tests_projection.py

```diff
@@ -0,0 +1,4 @@
+from KratosMultiphysics import *
```

Comment:
You don't need to create this file, the tests on KratosCoreFastSuite are run automatically

Reply:
ofc I will remove it, this is just for your convenience